pages/disputes/new.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div>
+    <h3> NOVO MÊS DE DISPUTA </h3>
+    <BorderCard
+      class="mt-10"
+      color="#00796b"
+    >
+      <v-text-field
+        label="NOME"
+        v-model="description"
+        required="true"
+      />
+      <v-btn
+        color="primary"
+        :disabled="!description || loading"
+        @click="saveDispute"
+      >
+        CRIAR
+      </v-btn>
+
+    </BorderCard>
+    <v-btn class="mt-10" to="/disputes" color="error"> Voltar </v-btn>
+  </div>
+</template>
+
+<script>
+import BorderCard from '@/components/shared/BorderCard'
+import gql from 'graphql-tag'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rafaelmbcosta/league-nuxt/pullRequest?prid=7487498)
```suggestion
import gql from "graphql-tag"
```